pkg/api/latest/group_register.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package latest
```

Comment:
This is an API machinery. It shouldn't live in the same package as pkg/api/latest/latest.go, which is group-specific. I think it's ok to move it to pkg/api/machinery/group_register.go for now, finally we want to move it to k8s.io/pkg/api-machinery/ (see this proposal: https://github.com/lavalamp/kubernetes/blob/types-doc/docs/proposals/api-types-package-structure.md#type-support-package-layout)

cc @lavalamp